index.html

```diff
@@ -15,12 +15,17 @@ <h1>Quiz-App</h1>
     <main>
       <section class="card">
         <p class="card__question">What property flips the axes in flexbox?</p>
-        <img class="card__bookmark" src="images/bookmark-icon-12323.png" />
+        <img
+          class="card__bookmark"
+          data-js="cardBookmark"
+          src="images/bookmark_FILL0_wght400_GRAD200_opsz40.svg"
+        />
```

Comment:
Since this will be clickable, please wrap an HTML button element around it, `img` elements per se should not be clickable.